composer.json

```diff
@@ -11,7 +11,7 @@
   "require": {
     "php": ">=5.4",
     "firebase/php-jwt": "~2.0|~3.0|~4.0|~5.0",
-    "guzzlehttp/guzzle": "^5.3.1|^6.2.1|^7.0",
+    "guzzlehttp/guzzle": "^6.5.8|^7.0",
```

Comment:
Can we stick to v7 already? If not just leave 6.5.8 alone

Reply:
I'm not sure who is using this repo. I prefer to keep 6.5.8 version OR 7

Reply:
Sure sure, I was just asking maybe we know more if not better not to break anything. Still I thing it just takes first version matching minimal requirements of other packages so it will always use 6.5.8. Because guzzle 7 requires php 7 it will be always omitted as we have "php": ">=5.4" . So again, if we aren't sure stick to lower one :)